src/components/PieChartFacetContents.js

```diff
@@ -17,10 +17,21 @@ type PieChartFacetContentsProps = {
    * the map are ignored.
    */
   entityColors: Map<string, string>;
+  /** Property to align the legend to left, right or center. */
+  legendAlign?: 'left' | 'right' | 'center';
+  /** Property to position the legend either as horizontal or vertical. */
+  legendLayout?: 'horizontal' | 'vertical';
+  /** Property to align the legend vertically either on top, middle or bottom. */
+  legendVerticalAlign?: 'top' | 'middle' | 'bottom';
+  /** Height of the Pie Chart. */
+  height?: number;
 };
 
 type PieChartFacetContentsDefaultProps = {
   entityColors: Map<string, string>;
+  legendAlign?: 'left' | 'right' | 'center';
+  legendLayout?: 'horizontal' | 'vertical';
+  legendVerticalAlign?: 'top' | 'middle' | 'bottom';
```

Comment:
You don't list the height property in the PieChartFacetContentsDefaultProps type definition.